synapse/config/appservice.py

```diff
@@ -30,12 +31,12 @@
 class AppServiceConfig(Config):
     section = "appservice"
 
-    def read_config(self, config, **kwargs):
+    def read_config(self, config, **kwargs) -> None:
         self.app_service_config_files = config.get("app_service_config_files", [])
         self.notify_appservices = config.get("notify_appservices", True)
         self.track_appservice_user_ips = config.get("track_appservice_user_ips", False)
 
-    def generate_config_section(cls, **kwargs):
+    def generate_config_section(cls, **kwargs) -> str:
```

Comment:
Do these methods ever do anything other than returning a string literal unconditionally?

I guess maybe one of them fills in a hostname or similar.

(I'm just wondering if this might be better off just being a static yaml file somewhere. Again, scope creep though.)

Reply:
Some of them return dynamic strings.